### ISSUE TITLE
Move Incomplete tasks only with a particular heading

### ISSUE THREAD

**Issue by build3r** [tgrosinger/slated-obsidian#58]
**Is your feature request related to a problem? Please describe.**
I have daily note which I create from a template, which has daily items section and a adhoc day to day to-do list (initially empty). I want only move the incomplete adhoc day to day items to the new daily not not all the items.

**Describe the solution you'd like**
Add an extra option like Move incomplete items from a certain section. Where the section can be a task list heading or pattern.

**Describe alternatives you've considered**
Manually move the incomplete tasks
I have modifies the code just to move the needed sections but it doesn't maintain the back links.
Modification:
```
const addTaskMove = (file, task, settings, vault, createLinks = true, moveChildren = true) => __awaiter(void 0, void 0, void 0, function* () {

  console.debug('Slated: Task heading: ' );
  console.debug(task.headings);
  if(task.headings[0].contains("## Tasks") == true) { //I need to only carry forward incomplete tasks from section which have heading starting with Tasks
    console.debug('Slated: Moving task to file: ' + file.basename);
    const linesToInsert = moveChildren ? task.subContent.slice() : [];
    linesToInsert.unshift(createLinks ? task.lineAsMovedFrom() : task.line);
    return withFileContents(file, vault, (lines) => {
        const taskSectionIndex = getIndexTasksHeading(lines, task.headings, settings);
        const taskSectionEndIndex = getIndexSectionLastContent(lines, taskSectionIndex);
        insertLines(lines, linesToInsert, taskSectionEndIndex + 1, settings);
        return true;
    });
  }
  else {
    console.debug('Slated: Not Moving ' + task.headings);
  }
});
```

**Additional context**
![image](https://user-images.githubusercontent.com/6481548/109330876-2cf0eb00-7882-11eb-80c7-37fa807a30ef.png)

**Comment by tgrosinger** [comment]
Hi @build3r, I understand your request and it seems reasonable, however I am trying to keep the number of commands and options in Slated as streamlined as possible. I'm going to think on this one to see if I can find a nice way to add this to the UI.

The pause will also give others a chance to express interest in this feature with a vote.

**Comment by tgrosinger** [comment]
I think this can be solved with #62
